src/Presentation/Nop.Web/Views/ShoppingCart/Wishlist.cshtml

```diff
@@ -112,14 +112,14 @@
                                         {
                                             <td class="remove-from-cart">
                                                 <label class="td-title">@T("ShoppingCart.Remove"):</label>
-                                                <input type="checkbox" name="removefromcart" value="@(item.Id)" />
+                                                <input type="checkbox" name="removefromcart" value="@(item.Id)" aria-label="@T("ShoppingCart.Remove")" />
                                             </td>
                                         }
                                         @if (Model.DisplayAddToCart)
                                         {
                                             <td class="add-to-cart">
                                                 <label class="td-title">@T("ShoppingCart.AddToCart"):</label>
-                                                <input type="checkbox" name="addtocart" value="@(item.Id)" />
+                                                <input type="checkbox" name="addtocart" value="@(item.Id)" aria-label="@T("ShoppingCart.Remove")" />

```

Comment:
At this point, the locale does not match the action

Reply:
I have removed the incorrectly placed aria-label from the PR